scala/scala.bzl

```diff
@@ -351,23 +379,58 @@ def collect_srcjars(targets):
             srcjars += [target.srcjars.srcjar]
     return srcjars
 
+def add_labels_of_jars_to(jars2labels, dependency, all_jars):
+  for jar in all_jars:
+   add_label_of_jar_to(jars2labels, dependency, jar)
+
+
+def add_label_of_jar_to(jars2labels, dependency, jar):
+ if label_already_exists(jars2labels, jar):
+   return
+
+ # skylark exposes only labels of direct dependencies.
```

Comment:
[issue](https://github.com/bazelbuild/bazel/issues/3367) was resolved, waiting for it to be officially released to switch to it